src/cargo/ops/registry/info/view.rs

```diff
@@ -1,7 +1,9 @@
 use std::collections::HashMap;
 use std::io::Write;
 
+use crate::core::FeatureValue;
 use crate::util::style::{ERROR, HEADER, LITERAL, NOP, NOTE, WARN};
+use crate::util::OptVersionReq;
```

Comment:
If we're doing a refactor like this, could you instead do it against `cargo-information`s repo to keep this PR minimal?

Reply:
Let me revert it. I think the old one also is OK.